scripts/question_answering/run_squad.py

```diff
@@ -777,7 +784,9 @@ def predict_extended(original_feature,
 def evaluate(args, last=True):
     ctx_l = parse_ctx(args.gpus)
     cfg, tokenizer, qa_net = get_network(
-        args.model_name, ctx_l, args.classifier_dropout)
+        args.model_name, ctx_l, args.classifier_dropout, dtype=args.eval_dtype)
+    if args.eval_dtype == 'float16':
+        qa_net.cast('float16')
```

Comment:
Is it possible to merge this into `get_network`? We may add tests in a later PR.

Reply:
in the case of training, I think we do not want to cast if eval_type is fp16, that is the reason to do it here.
In the coming days I think we can AMP for training as well